README.md

```diff
@@ -42,5 +42,5 @@ Example Usage
 ```sh
 # Download latest Kafka distribution(if necessary)
 $ tar zxvf kafka_LATEST_VERSION.tgz
-$ ./kafka_consumer_group_exporter --consumer-group-command-path=./kafka_LATEST_VERSION/bin/kafka-consumer-groups.sh BOOTSTRAP_SERVERS
+$ ./kafka_consumer_group_exporter --consumer-group-command-path=./kafka_LATEST_VERSION/bin/kafka-consumer-groups.sh BOOTSTRAP_SERVER
```

Comment:
Oh, I wanted to say here that I've never had any reason to use multiple servers here. The way we do scraping is that we are running the exporter on every Kafka broker. Each exporter is only connecting to localhost. That way, we can use the exporter to see if Kafka down on a specific broker while still getting HA since we are scraping all Kafka brokers. Your mileage might vary, but I thought this could be interesting for you to hear/know.